Dockerfile

```diff
@@ -1,8 +1,4 @@
-FROM letfn/python
-
-COPY --chown=app:app requirements.txt /app/src/
-RUN . /app/venv/bin/activate && pip install --no-cache-dir -r /app/src/requirements.txt
-
-COPY service /service
-
-ENTRYPOINT [ "/tini", "--", "/service" ]
+FROM alpine:edge
```

Comment:
The main Dockerfile is for kitt itself as a containerized tool.  That we need a second Dockerfile build suggests at minimum, the dnsmasq could be its own repository.  For example, look at letfn/pihole.  This separates building the image from using the image in a docker-compose.yml

Reply:
i see. so have the container already hosted in a docker repo. is there one available?

Reply:
Not yet.  Alternative is to move his to Dockerfile.dnsmasq and use it now.  Later extract to its own repo.